docs/_dg/Architecture.md

```diff
@@ -18,7 +18,7 @@ Given below is a quick overview of main components and how they interact with ea
 
 The rest of the App consists of four components.
 
-* [UI](#ui-component): Handles the User Interface of the App.
+* [UI](#ui-component): Handles the [[ graphical-user-interface:User Interface ]] of the App.
```

Comment:
Would it be better to use the full term "Graphical User Interface" here? Since we're linking to the entry for GUI. Alternatively, just remove the autoglossary.

Reply:
Ok let's remove then.